### PR TITLE
move formats pkg to and vendor from buildah

### PR DESCRIPTION
**Pull request by QiWang19**
move formats package to buildah/cmd/formats so buildah can use formats to solve images output format [#1239](https://github.com/containers/buildah/issues/1239)
PR in buildah uses formats pkg [#1398](https://github.com/containers/buildah/pull/1398)
Signed-off-by: Qi Wang <qiwan@redhat.com>

### CODE REVIEW

File: cmd/podman/diff.go [hunk 1]
```diff
@@ -3,7 +3,7 @@ package main
 import (
 	"fmt"
 	"github.com/containers/libpod/cmd/podman/cliconfig"
-	"github.com/containers/libpod/cmd/podman/formats"
+	"github.com/containers/buildah/cmd/formats"
```
Comment (rhatdan) [inline cmd/podman/diff.go]: Can we use them as a buildah/pkg/formats?

Comment (QiWang19) [reply cmd/podman/diff.go]: :ok_hand:

File: cmd/podman/system_df.go [hunk 2]
```diff
@@ -0,0 +1,408 @@
+package main
```
Comment (mheon) [inline cmd/podman/system_df.go]: Did this file sneak in by accident?

**Comment by mheon** [comment]
LGTM

File: cmd/podman/system_df.go (hunk 2 above)
Comment (QiWang19) [reply cmd/podman/system_df.go]: yes...

**Comment by rhatdan** [comment]
LGTM

**Comment by mheon** [comment]
Test failures seem like flakes

**Comment by vrothberg** [comment]
This PR has been merged to early as we're now vendoring a private branch for buildah instead of the official project. We should hurry and get https://github.com/containers/buildah/pull/1398 merged to get back to using github.com/containers/buildah.

Please mark such PRs as `WIP` to avoid accidentally merging them. I am currently blocked by it as I wanted to vendor it some new changed from buildah.

**Pull request merged**